spec.go

```diff
@@ -29,6 +29,8 @@ type Interface interface {
 	GetReleaseHistory(releaseName string) (*ReleaseHistory, error)
 	// InstallFromTarball installs a Helm Chart packaged in the given tarball.
 	InstallFromTarball(path, ns string, options ...helm.InstallOption) error
+	// RunReleaseTest runs the tests for a Helm Release.
+	RunReleaseTest(releaseName string, options ...helm.ReleaseTestOption) error
```

Comment:
Don't know if it was just me. But maybe make it a bit clearer in the godoc that we're actually using the helmclients 'RunReleaseTest' (equivalent to `> helm test` command) method here, and not just do some testing the release on our own?

Reply:
I think it was just me and I didn't read carefully enough

Reply:
I think putting it does similar job to the `helm test` wouldn't hurt.

Reply:
Good idea. I've updated the godoc.